pkg/controller/migmigration/postValidate.go

```diff
@@ -0,0 +1,298 @@
+package migmigration
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// EventDisplay is a structure for displaying occured events
+type EventDisplay struct {
```

Comment:
Why not just use the `core.Event`.  Also, the `Display` in the name describes a specific usage of the data instead of being descriptive of the data itself.  Suggest dropping the _display_ terminology.